appveyor.yml

```diff
@@ -1,24 +1,64 @@
 version: '{build}'
-image: Visual Studio 2019
+image:
+- Visual Studio 2019
+- Ubuntu1604
+- macos
+environment:
+  DOTNET_CLI_TELEMETRY_OPTOUT: true
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+for:
+-
+  matrix:
+    only:
+      - image: Visual Studio 2019
+  environment:
+    IMAGE_NAME: win
+  deploy:
+  - provider: NuGet
+    server: https://www.myget.org/F/morelinq/api/v2/package
+    api_key:
+      secure: fhGwXyO35FSshRzs5GWmF1LJTrd1sIqmS/jNCSfO2LfOciuYAKiXuFMYZFGiTAl+
```

Comment:
Isn't this supposed to be secret?

Reply:
Yes, but the secret is [secured](https://ci.appveyor.com/tools/encrypt) (as suggested in [Publishing to NuGet feed](https://www.appveyor.com/docs/deployment/nuget/)) and something only AppVeyor's build runner can decrypt.